charts/nri-bundle/README.md

```diff
@@ -25,6 +25,7 @@ here is a list of components that this chart installs and where you can find mor
 | [newrelic-prometheus-configurator](https://github.com/newrelic/newrelic-prometheus-configurator/tree/master/charts/newrelic-prometheus-agent) |  | Configures instances of Prometheus in Agent mode to send metrics to the New Relic Prometheus endpoint. |
 | [newrelic-pixie](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-pixie) |  | Connects to the Pixie API and enables the New Relic plugin in Pixie. The plugin allows you to export data from Pixie to New Relic for long-term data retention. |
 | [Pixie](https://docs.pixielabs.ai/installing-pixie/install-schemes/helm/#3.-deploy) |  | Is an open source observability tool for Kubernetes applications that uses eBPF to automatically capture telemetry data without the need for manual instrumentation. |
+| [nr-ebpf-agent](https://docs.pixielabs.ai/installing-pixie/install-schemes/helm/#3.-deploy) |  | Configures instances of the New Relic eBPF agent to automatically capture telemetry data without the need for manual instrumentation. |
```

Comment:
Will need to change the link to eBPF agent here.

Reply:
👍  Good catch.